API.md

```diff
@@ -0,0 +1,148 @@
+# API Documentation
+
+
+## Filters API
+
```

Comment:
we do not need this at all. we shouldnt have an APi for filters, we're using the kendra api